l10n/de.json

```diff
@@ -0,0 +1,22 @@
+{ "translations": {
+  "Social login": "Social login",
+  "Default group that all new users belong" : "Standardgruppe für neue Benutzer",
+  "None": "Keine",
+  "Save": "Speichern",
+  "Enabled": "Aktiviert",
+  "Disable auto create new users": "Automatische Erstellung neuer Benutzer deaktivieren",
+  "Unknown %s provider: \"%s\"": "Unbekannter %s Anbieter: \"%s\"",
+  "Auto creating new users is disabled": "Die automatische Erstellung neuer Benutzer is deaktiviert",
+  "This account already connected": "Dieses Konto ist bereits verbunden",
+  "Allow users to connect social logins with their account": "Benutzern erlauben externe Konten mit Ihrem Konto zu verknüpfen",
+  "Provider name cannot be empty": "Der Anbietername muss ausgefüllt sein",
+  "Duplicate provider name \"%s\"": "Der Anbietername \"%s\" existiert bereits",
+  "Invalid provider name \"%s\". Allowed characters \"0-9a-z_.@-\"": "Ungültiger Anbietername \"%s\". Es sind nur die folgenden Zeichen erlaubt \"0-9a-z_.@-\"",
+  "Social login connect": "Externe Konten verknüpfen",
+  "Available providers": "Verfügbare Anbieter"
```

Comment:
Missed comma

Reply:
Thanks. I've used the french translation files as a template. the comma has been added now.